landoscript/src/landoscript/github.py

```diff
@@ -0,0 +1,84 @@
+"""Treescript git functions."""
+
+import logging
+from string import Template
+from textwrap import dedent
+from typing import Dict, List, Optional, Union
+
+from simple_github import AppClient
+
+log = logging.getLogger(__name__)
+
+
+class UnknownBranchError(Exception):
+    pass
+
+
+class GithubClient:
```

Comment:
Maybe it's worth putting this in `scriptworker_client.github`? The slashes fix you incorporated here is evidence that it might be

Reply:
Good call, yeah, will do.

Reply:
I moved this part out to https://github.com/mozilla-releng/scriptworker-scripts/pull/1147.